aces/Loris
# [Media] Rename date_uploaded column to last_modified

## Brief summary of changes

The `Date Uploaded` column is not named accurately. It is a timestamp value that updates to the current timestamp upon edit. 

Given this behaviour, it should be called "Last Modified". This PR does that.

#### Testing instructions (if applicable)

1. Check out this pull request and run the patch
2. Run `make dev`
3. Go to the media module and verify that the column `Date Uploaded` now says `Last Modified`

#### Links to related tickets (GitHub, Redmine, ...)

* Resolves #5078
